src/main/java/org/cryptomator/frontend/fuse/mount/RevealException.java

```diff
@@ -0,0 +1,13 @@
+package org.cryptomator.frontend.fuse.mount;
+
+public class RevealException extends Exception{
+
+	public RevealException(String msg){
+		super(msg);
+	}
+
+	public RevealException(Throwable cause){
+		super(cause);
+	}
+
```

Comment:
Since this exception is intended to be thrown by third parties (from this library's point of view), add another constructor with Throwable _and_ Cause.

Reply:
Resolved in 16f648b